ddl/syncer.go

```diff
@@ -75,20 +104,41 @@ func (s *schemaVersionSyncer) Init(ctx goctx.Context) error {
 	if err != nil {
 		return errors.Trace(err)
 	}
-	s.GlobalVerCh = s.etcdCli.Watch(ctx, ddlGlobalSchemaVersion)
+	s.globalVerCh = s.etcdCli.Watch(ctx, ddlGlobalSchemaVersion)
 	return s.putKV(ctx, putKeyDefaultRetryCnt, s.selfSchemaVerPath, initialVersion)
 }
 
+// GlobalVersionCh implements SchemaSyncer.GlobalVersionCh interface.
+func (s *schemaVersionSyncer) GlobalVersionCh() clientv3.WatchChan {
+	return s.globalVerCh
+}
+
+// UpdateSelfVersion implements SchemaSyncer.UpdateSelfVersion interface.
 func (s *schemaVersionSyncer) UpdateSelfVersion(ctx goctx.Context, version int64) error {
 	ver := strconv.FormatInt(version, 10)
 	return s.putKV(ctx, putKeyNoRetry, s.selfSchemaVerPath, ver)
```

Comment:
It should put [with a lease option](https://godoc.org/github.com/coreos/etcd/clientv3#WithLease), so if `RemoveSelfVersionPath` fail, the path will be remove automatically.